util/Endpoint.php

```diff
@@ -158,18 +159,20 @@ public function renderClass(): string
                 : '\\' . $this->normalizeName($this->namespace),
             $class
         );
+        $class = isset($this->content['deprecation_message'])
+            ? str_replace(':deprecation-message', $this->content['deprecation_message'], $class)
+            : preg_replace('/\s*\/\*\*\s+\*\s+@deprecated :deprecation-message\s+\*\//', '', $class);
 
         // Set the HTTP method
         $action = $this->getMethod();
         if ($action === ['POST', 'PUT'] && $this->getClassName() !== 'Bulk') {
             $method = "'PUT'";
+        } elseif (!empty($this->content['body']) && ($action === ['GET', 'POST'] || $action === ['POST', 'GET'])) {
```

Comment:
You can sort $action instead of the check, which will work better if you need to add another method (combinations of 3 are harder :)).